modules/exploits/linux/http/webmin_packageup_rce.rb

```diff
@@ -0,0 +1,154 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Webmin <= 1.910 - "Package Updates" Remote Command Execution',
```

Comment:
Check the name field guide here: https://github.com/rapid7/metasploit-framework/wiki/How-to-get-started-with-writing-an-exploit
I would suggestion: `Webmin Package Updates Remote Command Execution`